e_os.h

```diff
@@ -7,8 +7,8 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_E_OS_H
-# define HEADER_E_OS_H
+#ifndef E_OS_H
+# define E_OS_H
```

Comment:
That's kinda exceptional...  I would have expected `OSSL_E_OS_H`...

Reply:
Oh, that's because it is in the root of the source tree! I overlooked this case. I'll fix that.

Reply:
Fixed by 9a2e0cd94be9b2edd9efa5629aacf017c4ebc8cb, for the result see [force-push](https://github.com/openssl/openssl/compare/8088e6bfd65fb8be72e3c1b85850800b160b35bc..31c521f705cc10ab1e0477d9693cbdc374834885) diff.